src/OpenTelemetry.ClrProfiler.Native/environment_variables.h

```diff
@@ -77,17 +77,17 @@ const WSTRING azure_app_services_cli_telemetry_profile_value =
     WStr("DOTNET_CLI_TELEMETRY_PROFILE");
 
 // Enable the profiler to dump the IL original code and modification to the log.
-const WSTRING dump_il_rewrite_enabled = WStr("OTEL_DUMP_ILREWRITE_ENABLED");
+const WSTRING dump_il_rewrite_enabled = WStr("OTEL_DOTNET_AUTO_DUMP_ILREWRITE_ENABLED");
 
 // Sets whether to enable JIT inlining
-const WSTRING clr_enable_inlining = WStr("OTEL_CLR_ENABLE_INLINING");
+const WSTRING clr_enable_inlining = WStr("OTEL_DOTNET_AUTO_CLR_ENABLE_INLINING");
 
 // Custom internal tracer profiler path
 const WSTRING internal_trace_profiler_path =
```

Comment:
BTW in a later PR this specific variable can be removed, it's not used anywhere in this repo

Reply:
Sorry, I was referring to `internal_trace_profiler_path`, not `clr_enable_inlining`

Reply:
Let's do it in a separate PR.

Reply:
Issue created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/360